tests/dummy/app/controllers/freestyle.js

```diff
@@ -273,6 +273,18 @@ export default FreestyleController.extend(ModalMixin, {
       buttonOneType: 'secondary',
     }),
 
+
+    this.set('modalNotificationParams', {
+      closeAction: this.controllerCloseAction,
+      notificationModal: true,
+      notificationIcon: 'bourbon-confirm-email-icon',
+      title: 'Please confirm your email',
+      textContent: 'We just sent you a link to click on to confirm your email address. Please confirm this as soon as you can.',
+      buttonOneTitle: 'Sounds good',
+      buttonOneAction: this.primaryClick,
+    }),
```

Comment:
Dammit, Microsoft. There's still a tiny part of my brain saying "that `,` at the end of the last entry of the hash will cause IE6 to die!"

Reply:
Stoopid Microsoft and stoopid IE6!!! 

I can update this but this file is really only for our examples and things being used inside our documentation and examples, so really shouldn't be affected by their stupidity.  :)